pkgs/applications/misc/immersed-vr/default.nix

```diff
@@ -0,0 +1,23 @@
+{pkgs ? import <nixpkgs> {}, fetchurl, lib }:
+pkgs.appimageTools.wrapType2 {
+  pname = "immersed-vr";
+  version = "9.6";
+
+  src = fetchurl {
+    url = "https://static.immersed.com/dl/Immersed-x86_64.AppImage";
+    hash = "sha256-iA0SQlPktETFXEqCbSoWV9NaWVahkPa6qO4Cfju0aBQ=";
+  };
+
+  extraPkgs = pkgs: with pkgs; [
+    libthai
+  ];
+
+  meta = with lib; {
```

Comment:
Please add `meta.homepage`.
It would be helpful if you could also post a brief description and url in the pull request summary.